src/go/rpk/pkg/tuners/irq/cpu_masks.go

```diff
@@ -83,6 +84,21 @@ func (masks *cpuMasks) CPUMaskForComputations(
 	} else if mode == Mq {
 		// all available cores
 		computationsMask = cpuMask
+	} else if mode == Dedicated {
```

Comment:
`CPUMaskForComputations` ... computations? This is getting the mask for the cores redpanda should run on?